packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -168,6 +214,15 @@ const FinancialDetails = props => {
                                                 other_instruments_trading_frequency_enum={
                                                     props.other_instruments_trading_frequency_enum
                                                 }
+                                                income_source_enum={[]}
```

Comment:
This is weird :smile: Can we handle them inside the component instead of writing them like this?

Reply:
how do you mean?

Reply:
Make these properties optional and handling undefined properties inside the component itself

Reply:
I can do that for this component but the whole file needs refactoring. other ones are like this as well.so maybe it's better to handle it in a separate card. what do you think?

Reply:
Alrighty shayan, that's fine